crates/oxc_parser/src/lexer/token.rs

```diff
@@ -53,7 +53,7 @@ impl Default for Token {
         // end: 0,
         // kind: Kind::default(),
         // is_on_new_line: false,
-        // escaped: false,
+        // escape_index: 0,
```

Comment:
Nit: Please move this down to bottom of the list, so the list has same order as the fields in `Token`.